microSALT/cli.py

```diff
@@ -120,6 +120,9 @@ def root(ctx):
 @click.option(
     "--skip_update", default=False, help="Skips downloading of references", is_flag=True
 )
+@click.option(
+    "--force_update", default=False, help="Forces downloading of pubMLST references", is_flag=True
```

Comment:
Skip and force could be on/off for the same flag.

Reply:
Nevermind. 
Doing that means the default value for the flag has to be NULL. Which will cause confusion.